modules/realvuBidAdapter.js

```diff
@@ -0,0 +1,280 @@
+import { getBidRequest } from 'src/utils';
+import adaptermanager from 'src/adaptermanager';
+
+const CONSTANTS = require('src/constants');
+const utils = require('src/utils.js');
+const adloader = require('src/adloader.js');
+const bidmanager = require('src/bidmanager.js');
+const bidfactory = require('src/bidfactory.js');
+const Adapter = require('src/adapter.js').default;
+// var AppnexusAdapter = require('./appnexusBidAdapter.js');
+
+var RealVuAdapter = function RealVuAdapter() {
+  var baseAdapter = new Adapter('realvu');
+  baseAdapter.callBids = function (params) {
+    // utils.logMessage('realvuBidAdapter params: ' + JSON.stringify(params));
+    var pbids = params.bids;
+    var boost_back = function() {
+      var top1 = window;
+      realvu_frm = 0;
+      try {
+        var wnd = window;
+        while ((top1 != top) && (typeof (wnd.document) != 'undefined')) {
+          top1 = wnd;
+          wnd = wnd.parent;
+        }
+      } catch (e) { };
+      for (var i = 0; i < pbids.length; i++) {
+        var bid_rq = pbids[i];
+        var sizes = utils.parseSizesInput(bid_rq.sizes);
+        top1.realvu_boost.addUnitById({
+          partner_id: bid_rq.params.partnerId,
+          unit_id: bid_rq.placementCode,
+          callback: baseAdapter.boostCall,
+          pbjs_bid: bid_rq,
+          size: sizes[0],
+          mode: 'kvp'
+        });
+      }
+    };
+    adloader.loadScript('//ac.realvu.net/realvu_boost.js', boost_back, 1);
+  };
+
+  baseAdapter.boostCall = function(rez) {
+    var bid_request = rez.pin.pbjs_bid;
+    var callbackId = bid_request.bidId;
+    // utils.logMessage('realvuBidAdapter boost callback "' + callbackId + '", rez.realvu=' + rez.realvu);
+    if (rez.realvu === 'yes') {
+      var adap = new RvAppNexusAdapter();
+      adloader.loadScript(adap.buildJPTCall(bid_request, callbackId));
+    } else { // not in view - respond with no bid.
+      var adResponse = bidfactory.createBid(2);
+      adResponse.bidderCode = 'realvu';
+      bidmanager.addBidResponse(bid_request.placementCode, adResponse);
+    }
+  };
+
+  // +copy/pasted appnexusBidAdapter, "handleAnCB" replaced with "handleRvAnCB"
+  var RvAppNexusAdapter = function RvAppNexusAdapter() {
+    var baseAdapter = new Adapter('appnexus');
+    var usersync = false;
+
+    baseAdapter.callBids = function (params) {
```

Comment:
Are you using this callBids anywhere ?  I think its not getting used, you can remove it.